mentor-notes.md

```diff
@@ -1,48 +0,0 @@
----
-description: >-
-  Overview of how these workshops should be facilitated and delivered and how
-  participants should be grouped.
----
-
-# Facilitator Notes
-
-## Facilitators
-
-Each workshop should be run by two facilitators who take turn in presenting the content and exercises.
-
-## Participants
-
-We recommend delivering this curriculum to groups of two or three charities. In our experience, larger group sizes compromise the collaborative nature of the workshops. Each charity should be represented by one person who embraces the role of Product Manager within their organisation.
-
-We found that previous participants particularly valued the collaborative nature of the workshops. Therefore, charities should be grouped in a way that encourages cooperation between participants and leverages the collaborative nature of the curriculum. To this end, we propose to consider the following criteria when grouping participating charities:
-
-**Primary grouping criteria**
-
-- Shared interest/problem area
-- Similar progress status within the product development
-
-**Secondary grouping criteria**
-
-- Shared level of commitment
-- Mixed level of expertise\*
-- Availability & location
-
-**\*** Groups with mixed levels of experience can provide an opportunity for lesser experienced charities to learn from more experienced charities.
-
-## Facilities
-
-We found that the workshop space \(size and layout\) impacts level of collaboration/dialogue between charities. Therefore, we recommend delivering the workshops in a room that allows the participants to move and communicate freely. We also suggest that the room has a projector or a large screen as well as a whiteboard or flip chart. If this is not available, we recommend that the facilitators bring large white paper for the exercises.
-
-Facilitators should also bring pens and post-it notes to each workshop.
-
-## **Communication**
-
-In larger cohorts, communication should be handled via Slack (or a similar instant messaging platform). This provides quick and clear communication that is advantageous to communicating via email.
-
-Participants should be informed about the agenda of each workshop in advance so that they know what to expect and what to prepapre.
-
-## Feedback
-
-During the programme, organisations should be given opportunities to give feedback to facilitators. This aids mutual understanding and allows the programme's pace to be adjusted to the pace of participants. The last 15 minutes of each workshop should be dedicated to questions and check-out.
-
-This also encourages participants to act as a cohort and to feel in charge of the process.
```

Comment:
This file could be the basis of the top-level facilitator notes - with a decent overhaul.

Reply:
Wanted to discuss this with you. I have made a space in readme for facilitator notes as agreed but maybe we want a whole page for it - these notes are saved on my comp so we can add best bits in either way